dockyard/api/app.py

```diff
@@ -0,0 +1,42 @@
+import pecan
+
+from dockyard.api import config as api_config
+from oslo_config import cfg
+from pecan import make_app
+
+from dockyard import model
+
+
+API_SERVICE_OPT = [
+    cfg.PortOpt('port',
+                 default=5869,
+                 help='Port for the dockyard service.'),
+    cfg.IPOpt('host',
+              default='127.0.0.1',
+              help='Listening address for dockyard service'),
+]
+
+CONF = cfg.CONF
+opt_group = cfg.OptGroup(name='default',
+                         title='Group for the default values for dockyard api')
+CONF.register_group(opt_group)
+CONF.register_opts(API_SERVICE_OPT, opt_group)
+
+def get_pecan_config():
+    # Set up the pecan configuration
+    filename = api_config.__file__.replace('.pyc', '.py')
+    return pecan.configuration.conf_from_file(filename)
```

Comment:
I think specific import can be done here, from pecan import configuration, need to check any guideline, use better and consistent.